05-Time-Series-Raster.Rmd

```diff
@@ -213,8 +205,32 @@ xres(NDVI_HARV_stack)
 
 ```
 
+Notice that the CRS is `+proj=utm +zone=19 +ellps=WGS84 +units=m +no_defs`. The
+**CRS** is in UTM Zone 19.  If you have completed the previous tutorials in 
+this 
+[raster data in `R` series ]({{ site.baseurl }}tutorial/spatial-raster-series),
```

Comment:
Please edit:

you may have noticed that the UTM zone for the NEON collected remote sensing data was in Zone 18 rather than Zone 19. Why are the Landsat data in Zone 19?